### DISCUSSION TITLE
Forumoji picker for the Scratch Forums!

### DISCUSSION THREAD

**Discussion by RedToaster55** [ScratchAddons/ScratchAddons#7452] [Ideas] [answered]
I've been using [forumoji](https://scratch.mit.edu/discuss/topic/557083/) quite a bit recently and I thought it'd be nice if the native forum emoji picker were upgraded to a scroll-able selection of forumojis as opposed to the limited selection currently available.
(I'll work on a mock-up.)

**Comment by Samq64** [comment, marked as answer]
Duplicate of #4097.

**Reply by RedToaster55** [reply]
Why didn't that come up in my search for "forumoji"?? Sorry about that...
